lib/ts/recipe/thirdpartyemailpassword/emaildelivery/services/backwardCompatibility/index.ts

```diff
@@ -47,7 +46,7 @@ export default class BackwardCompatibilityService
         let emailVerificationFeatureNormalisedConfig =
             inputCreateAndSendCustomEmail !== undefined
                 ? {
-                      createAndSendCustomEmail: async (user: EmailVerificationUser, link: string, userContext: any) => {
+                      createAndSendCustomEmail: async (user: EmailPasswordUser, link: string, userContext: any) => {
```

Comment:
- Add a test to sign up with third party and then send them an email verification email (this should fail)
- We have to create an email verification backwards compatibility service as well (and then the above test should pass).